cell_type_prediction/galaxy_cli_use_case/cell_types_prod_control_wf.yaml

```diff
@@ -0,0 +1,1897 @@
+{
```

Comment:
I'm guess that you might want to delete this or the other file with the same name, and or change the extension of this to .json, as it is not a yaml.